pkg/docker-ce/Dockerfile

```diff
@@ -17,10 +17,10 @@ RUN apk add --no-cache --initdb -p /out \
 	xz
 RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
 
-FROM linuxkit/vpnkit-forwarder:e2776b82ddfe82ed7f90e55d7a2b424e62e9a279 AS vpnkit
+FROM linuxkit/vpnkit-expose-port:40089cdacba246d9d72b3f507640ef9c8e778838 AS vpnkit
```

Comment:
`docker-ce` doesn't actually need `vpnkit-expose-port` after https://github.com/linuxkit/linuxkit/pull/2124. It's now bind mounted from the init package `pkg/vpnkit-expose-port` in the blueprint instead: https://github.com/linuxkit/linuxkit/blob/master/blueprints/docker-for-mac/docker-17.06-ce.yml#L20